tests/system/src/test/java/test/robot/javafx/embed/swing/SwingNodePlatformExitCrashTest.java

```diff
@@ -53,6 +54,7 @@ public void testPlatformExitBeforeShowHoldEDT() throws InvocationTargetException
 
     @BeforeAll
     public static void skipShutDown() {
+        Assumptions.assumeTrue(!Util.isOnWayland());
```

Comment:
Please add `// JDK-8350009` in a comment, either at the end of this line or on a line by itself before this).